### PR TITLE
Features/#423 change schmidt industrial sites

### LINKED ISSUE #423: change demand.egon_schmidt_industrial_sites to full Schmidt's dataset
The table egon_schmidt_industrial_sites contains industrial sites with DSM-potential. 
It is derived of the data gathered for Danielle Schmidt's master thesis: https://zenodo.org/record/3613767#.YS9xcVtCS-p

For now, duplicate entries containing the same plants, but different applications get dropped. 
For modelling the DSM-potentials, those additional rows are relevant as these describe the applications of the plants. 
Therefore, the table should be integrated with this additional information.

### PR DESCRIPTION
**Pull request by KathiEsterl**
The industrial sites gathered by Danielle Schmidt's (https://zenodo.org/record/3613767#.YS9xcVtCS-p) are getting imported as they are, plants with different applications will no longer be dropped, because this information is important for the modelling of DSM.
Closes #423

### CODE REVIEW

**Comment by KathiEsterl** [comment]
As Ilka won't be able to check those changes during the next days, maybe you can have a look on it please, Clara? Thank You!

**Comment by ClaraBuettner** [comment]
I'm wondering what I should review. Should I only confirm that the tasks are running and the code looks fine or should I also have a look at the industrial sites table?

**Comment by KathiEsterl** [comment]
> I'm wondering what I should review. Should I only confirm that the tasks are running and the code looks fine or should I also have a look at the industrial sites table?

I have checked the tables several times for SH and whole Germany, I think they should be fine and you don't need to review them. So it would probably be just confirming that the tasks are running and that the code looks fine, I guess.

File: src/egon/data/datasets/industrial_sites/__init__.py [hunk 1]
```diff
@@ -689,7 +680,7 @@ class MergeIndustrialSites(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="Merge_industrial_sites",
-            version="0.0.1",
+            version="0.0.2",
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/industrial_sites/__init__.py:683]: The version number was already updated to 0.0.2 on the CI-branch. Could you please update the version number to 0.0.3 to be sure it is also updated on the CI-branch?

**Comment by ClaraBuettner** [comment]
I run the branch successfully and had a look at the code. I just had one comment regarding the version number. When this is fixed we can merge it into to CI-branch.

**Comment by KathiEsterl** [comment]
> I run the branch successfully and had a look at the code. I just had one comment regarding the version number. When this is fixed we can merge it into to CI-branch.

Thank you! I have changed the version.

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**